API Blueprint Specification.md

```diff
@@ -478,6 +490,232 @@ Name and description of the API
 
 ---
 
+<a name="def-authentication-section"></a>
+## Authentication section
+- **Parent sections:** none
+- **Nested sections:** [`1` Authentication schemes section](#def-authenticationschemes-section), [`1` Response section](#def-response-section)
+- **Markdown entity:** header
+- **Inherits from**: none
+
+#### Definition
+Defined by the `Authentication` keyword in Markdown header entity.
+
+    # Authentication
```

Comment:
Technically, I think API Blueprint parser ignores levels of headers and by subsequent tooling they get deduced implicitly from the semantic hierarchy in AST. Not sure though. I think it works like that inside Apiary - it's always rendered the same regardless the actual level of the header.

Anyway, I agree it would be better to indicate a sub-header in the example not to confuse anyone.